src/backend/revoker/scanAttestations.ts

```diff
@@ -22,21 +16,44 @@ export interface AttestationInfo extends Omit<IAttestation, 'revoked'> {
 
 let fromBlock = 0;
 
+/** Extends the `event` with the parameters parsed,
+ *  so that the parameters value extraction is easier and more elegant.
+ *
+ * @param event
+ * @returns the extended event
+ */
+function paramsParser(event: ParsedEvent) {
```

Comment:
It’s a good practice to name functions with a verb, reserving nouns for objects

Reply:
Changed in: [c4c8371](https://github.com/BTE-Trusted-Entity/socialkyc.io/pull/1957/commits/c4c837189371e8d43a9d0c14de7e4b369dfa19a4)